src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static String color;
+
+    public static Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/5c6396c3-8269-48a8-8c06-8ad5e75fc5e1)

Reply:
not fixed